src/components/ClassCard/ClassCard.css

```diff
@@ -0,0 +1,54 @@
+.ClassCard__wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  width: 100%;
+}
+
+.ClassCard {
+  background-color: var(--color-text-onDark-dim);
+  color: var(--color-primary);
+  padding: 1rem;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.ClassCard h2 {
+  padding-bottom: 1rem;
+}
+
+.ClassCard:active,
+.ClassCard:hover {
+  background-color: var(--color-accent-onAction);
+  color: var(--color-text-onDark);
+  transition: all 1s ease-in-out;
```

Comment:
Nice transition effect, but for my taste 1s is too long. Maybe have only 0.4s🤔

Reply:
Nice catch!